plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/api/Menu.java

```diff
@@ -42,6 +42,16 @@
 	 * @return menu text
 	 */
 	String getText();
+	
+	/**
+	 * Returns label of the menu.
+	 * i.e. Edit -> Content Assist -> Default
+	 * 		- text: "&Default\tCtrl+Space"
+	 * 		- label: "Default"
+	 * 
+	 * @return menu text
+	 */
+	String getLabel();
```

Comment:
I dont think this is needed. existing getText() should be used instead and it should return text without "&" symbol